crt_portal/cts_forms/templates/forms/complaint_view/index/complaints_table.html

```diff
@@ -1,136 +1,180 @@
-        <td></td>
-        <td colspan="2">
-          <div class="td-quickview">
-            Contact
-          </div>
-          <div class="display-flex flex-column">
-            <span>
-              {{ datum.report.contact_last_name|default:"-" }} {{ datum.report.contact_first_name|default:"-" }}
-            </span>
-            <span>
-              {{ datum.report.contact_email|default:"-" }}
-            </span>
-            <span>
-              {{ datum.report.contact_phone|default:"-" }}
-            </span>
-          </div>
-        </td>
-        <td colspan="4">
-          <div class="td-quickview">
-            Personal Description
-          </div>
-          <div class="td-summary">
-            {{ datum.report.violation_summary|linebreaks|default:"-" }}
-          </div>
-        </td>
-        <td colspan="2"></td>
-      </tr>
-    {% endfor %}
-    </tbody>
-  {% endif %}
-</table>
-</div>
+<form class="usa-form"
+      method="get"
+      action="{% url 'crt_forms:crt-forms-actions' %}"
+      novalidate
+>
+  <input type="hidden" value="{{ return_url_args }}" name="next" id="next" />
+
+  <div class="crt-xscroll">
+    <table class="usa-table crt-table sort-table">
+      <thead>
+        <tr>
+          <th>
+            <div class="usa-checkbox td-checkbox">
+              <input type="checkbox"
+                     name="all"
+                     class="usa-checkbox__input"
+                     value="all"
+                     id="checkbox-all"
+              >
+              <label class="crt-checkbox__label" for="checkbox-all"></label>
+            </div>
+          </th>
+          <th></th>
+          {% render_sortable_heading 'Status' sort_state filter_state %}
+          {% render_sortable_heading 'Routed' sort_state filter_state %}
+          {% render_sortable_heading 'Submitted' sort_state filter_state %}
+          {% render_sortable_heading 'Contact Name' sort_state filter_state %}
+          {% render_sortable_heading 'Contact Details' sort_state filter_state %}
+          {% render_sortable_heading 'Incident Location' sort_state filter_state %}
+          {% render_sortable_heading 'Summary' sort_state filter_state %}
+          {% render_sortable_heading 'Incident Date' sort_state filter_state %}
+        </tr>
+      </thead>
+      {% if data_dict %}
+        <tbody>
+          {% for datum in data_dict %}
+            <tr class="tr-status-{{ datum.report.status }} tr--hover{% cycle ' stripe' '' %}">
+              <td>
+                <div class="usa-checkbox td-checkbox">
+                  <input type="checkbox"
+                         name="id"
+                         class="usa-checkbox__input"
+                         value="{{ datum.report.id }}"
+                         id="checkbox-{{ datum.report.id }}"
+                  >
+                  <label class="crt-checkbox__label" for="checkbox-{{ datum.report.id }}"></label>
```

Comment:
Ooo, this generates an a11y scanning failure I think due to the label itself it being empty.